modules/ROOT/pages/reference/procedures.adoc

```diff
@@ -965,18 +965,8 @@ m|READ
 |===
 | Description
 a|
-Query the given full-text index.
-
-Returns the matching nodes and their Lucene query score, ordered by score.
-
-Valid _key: value_ pairs for the `options` map are:
-
-* `skip: <number>` -- skip the top N results.
-* `limit: <number>` -- limit the number of results returned.
-* `analyzer: <string>` -- use the specified analyzer as a search analyzer for this query.
-
-The `options` map and any of the keys are optional.
-An example of the `options` map: `{skip: 30, limit: 10, analyzer: 'whitespace'}`
+Query the given full-text index. Returns the matching nodes, and their Lucene query score, ordered by score.
+Valid keys for the options map are: 'skip' to skip the top N results; 'limit' to limit the number of results returned; 'analyzer' to use the specified analyzer as search analyzer for this query.
```

Comment:
```suggestion
Valid _key: value_ pairs for the `options` map are: 

* `skip: <number>` -- to skip the top N results. 
* `limit: <number>` -- to limit the number of results returned. 
* `analyzer: <string>` -- to use the specified analyzer as a search analyzer for this query.
```

Reply:
I think, that's the best way to represent a list. The question is: is it possible to repeat this formatting in the code? Do we have to do it to prevent docs tests from failing?

Reply:
the answer is that we ca use the same asciidoc in the code